### DISCUSSION TITLE
Does it support 'right click' - for menu opening

### DISCUSSION THREAD

**Discussion by lrauhockey** [Skyvern-AI/skyvern#974] [Q&A]
Just wondering if it supports right click, or can understand if need to say right click on a row or element to be able to do a function. Or does it understand only basic form actions.

**Comment by lwc-alex** [comment]
I also want to use the browser developer tool like console or inspect, and it failed with
"The user goal involves using browser developer tools, which cannot be controlled through webpage elements."

**Comment by suchintan** [comment]
What did you want to do after the right click? Can it be simulated with a left click instead? (ie is there a menu button it can click on?
